### PR TITLE
Implementing new UI for sign up/sign in

### LINKED ISSUE #599: Implement new login/signup UI
@vazquez - let's separate this from age verification. Add annotations here. No need to change mocks for age verification. 

Note: on login view - let's add "username or email" to first field's label. Both currently work for login.

### PR DESCRIPTION
**Pull request by vazquez**
Fixes #599

### CODE REVIEW

**Comment by vazquez** [comment]
### Sign Up

screen shot 2015-08-26 at 16 21 08

---

### Sign In

screen shot 2015-08-26 at 16 21 13

**Comment by vazquez** [comment]
R? @k88hudson :rainbow:

File: src/pages/login/login.less [hunk 1]
```diff
@@ -62,40 +89,49 @@
...
     font-size: 20px;
     transition: all 0.2s ease-in-out;
-    &:focus {
-      box-shadow: 0 2px 0 0 @white;
-    }
+    &::placeholder{
+      color: fade(@white, 50%);
+      font-size: 14px;
+      opacity: 1;
+    };
```
Comment (k88hudson) [inline src/pages/login/login.less]: extra semicolon

File: src/pages/login/login.less [hunk 2]
```diff
@@ -62,40 +89,49 @@
   input[type="email"] {
     background: transparent;
     border: none;
-    box-shadow: 0 1px 0 0 fade(@white, 50%);
     font-size: 20px;
     transition: all 0.2s ease-in-out;
-    &:focus {
-      box-shadow: 0 2px 0 0 @white;
-    }
+    &::placeholder{
```
Comment (k88hudson) [inline src/pages/login/login.less]: space after `placeholder`

File: src/pages/login/login.less [hunk 3]
```diff
@@ -53,8 +66,22 @@
...
+    label, input {
+      width: 85%;
+      display: block;
+      margin: 12px auto;
+    }
+    .form-group {
+      margin: 0;
+      border-bottom: 2px solid fade(@white, 15%);
+    }
+    >div:nth-of-type(1){
```
Comment (k88hudson) [inline src/pages/login/login.less]: space after `nth-of-type(1)`.

Also, why not `first-of-type`?

**Comment by k88hudson** [comment]
A couple of small style nits, but otherwise this look amazing!

One fairly big issue though -- I think it's an A++ idea to allow people to sign in with either username or email, but we will need to actually add some logic in to support this

**Comment by k88hudson** [comment]
Personally, I think doing that would be better left as a follow-up ticket. What do oyu think?

**Comment by vazquez** [comment]
@k88hudson I'm leaving the `username or email` for a follow-up ticket :) I'll revert back to just `username` for now.

File: src/pages/login/login.less (hunk 3 above)
Comment (vazquez) [inline src/pages/login/login.less]: :+1:!

**Comment by vazquez** [comment]
giphy 10 12 38 am

**Pull request merged**